proposals/incubation/credebl.md

```diff
@@ -0,0 +1,127 @@
+
+Once these final steps are completed, we believe the project will be ready for "Graduated" status and will apply for
+this designation immediately.
+
+We anticipate that this designation process will proceed swiftly if the TAC concurs that all criteria have been met.
+
+
+# Solution
+The CREDEBL Codebase is currently hosted as a separate Github Org at https://github.com/credebl
+
+### CREDEBL PLATFORM
+Platform provides a set of scalable services to manage decentralized identity and verifiable credentials (VC) system. It enables developers to build Self-Sovereign Identity (SSI) solutions that provide enhanced privacy and security. It is designed for scalability to support population-scale SSI implementations. 
+
+It provides tools for digital credential management and verification, allowing organizations to easily issue and manage credentials with ease. It is built on a micro-services architecture to support large-scale enterprise applications.
+
+Platform supports multiple ledger options (like Indy and Polygon) and also ledger-less credential issuance using methods like did:web, did:key and did:peer.
+
+### CREDEBL STUDIO
+Studio is web front-end for the entire Platform, built using Astro, React, Flowbite, and Tailwind CSS, allowing administrators & users to easily access & manage all the Platform features using a simple web interface.
+
+### ARIES AGENT
+In the context of decentralized identity systems, an agent generally refers to a software component or service that facilitates peer-to-peer interactions and data exchange operations between various entities in the system.
+
+Agents leverage the Aries protocol implementations built into Credo & Aca-Py.
+
+There are two types of agents that organizations can use, each serving different needs based on the deployment model and use case: Shared Agents and Dedicated Agents.
+
+A shared agent refers to an agent that is used by multiple entities or tenants, rather than being dedicated to a single entity. If the tenancy is set to true, it typically means that the system or agent supports multi-tenancy, or if the tenancy is set to false, it means that the agent does not support the multi-tenancy.
+
+### Mobile SDK
+Built on the strong foundation of Credo, the Mobile Wallet SDK is a React-Native SDK helps mobile developers to easily build SSI features with into existing mobile apps or a new one, enables interoperability and seamless connectivity for secure data exchange.
+
+# Architecture
+Following diagram depicts the overall system architecture of CREDEBL
+
+<img src="https://docs.credebl.id/~gitbook/image?url=https%3A%2F%2F292774163-files.gitbook.io%2F%7E%2Ffiles%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fspaces%252FnsdydI5ziLfZ8yT3e0r8%252Fuploads%252FXuF6F26u8f2KJS0NsD7p%252Fimage.png%3Falt%3Dmedia%26token%3Db7e5808c-ce94-4dae-b0d2-4ae0a3aa8954&width=768&dpr=4&quality=100&sign=dcaff92c&sv=1">
```

Comment:
It would be good to understand from this diagram, which pieces are considered in scope of the contribution? What are the dependencies to other projects?

Reply:
Okay, I will post a new diagram, clearly showing the pieces considered for contribution vs. the ones which are not.

Reply:
Updated with new diagram.